obspy/imaging/beachball.py

```diff
@@ -351,34 +355,58 @@ def plot_mt(T, N, P, size=200, plot_zerotrace=True,  # noqa
             colors.append('w')
         return colors, collect
 
-    if np.fabs(v[0]) >= np.fabs(v[2]):
+    if (plot_zerotrace):
```

Comment:
not convinced () are needed for if statements, have to check with flake8 results